cypress/e2e/topNavigationMobile.cy.ts

```diff
@@ -0,0 +1,97 @@
+/// <reference types='cypress' />
+
+Cypress.config("viewportWidth", 990);
+
+import "cypress-real-events";
+import { checkIfElementIsEntirelyInViewport } from "../utils";
+
+describe("Top navigation in mobile view (below 992px width) test", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.get('.navbar a[href="/"]').as("home");
+    cy.get('.navbar a[href="/projects"]').as("projects");
+    cy.get('.navbar a[href="/talks"]').as("talks");
+    cy.get('.navbar a[href="/blog"]').as("blog");
+    cy.get('.navbar a[href="#hire-me"]').as("contact");
+  });
+
+  it("Should open and close top navigation with toggler icon on every page", () => {
+    ["Home", "Projects", "Talks", "Blog", "Contact"].forEach((item) => {
+      cy.get("#navigation").should("have.class", "collapse");
+      cy.get(".navbar-toggler-icon").click();
+      cy.get("#navigation").should("not.have.class", "collapse");
+      cy.get(`@${item.toLowerCase()}`).click();
+      cy.get("#navigation").should("have.class", "collapse");
+    });
+  });
+
+  it("Should check if top navigation buttons have correct addresses", () => {
+    cy.get(".navbar-toggler-icon").click();
+    ["Home", "Projects", "Talks", "Blog", "Contact"].forEach((item) => {
+      cy.get(`@${item.toLowerCase()}`).should("include.text", item);
+    });
+  });
+
+  context("Navigation flow", () => {
+    function goBackToHome() {
+      return cy.go("back").location("pathname").should("eq", "/");
+    }
```

Comment:
It looks exactly like function used in topNavigation.cy.ts.

Instead of repeating this, I would suggest to move this into `cypress/utils/index.ts`, so this could be imported and use here